Note that `core.hash_node_position` is not a hash function (#15835)
luanti-org/luanti: doc/lua_api.md

    * This function can be overridden by mods to change the leave message.
* `core.hash_node_position(pos)`: returns a 48-bit integer
    * `pos`: table {x=number, y=number, z=number},
    * Gives a unique numeric encoding for a node position (16+16+16=48bit)

>     * Despite the name, this does not have collisions and this is not a mixing
>       hash function, neighboring x positions will have consecutive values.

Reviewer: ```suggestion
    * Despite the name, this is not a hash function (so it doesn't mix or produce collisions).
```
